### ISSUE TITLE
What Edition?

### ISSUE THREAD

**Issue by KWexoo** [rounakbanik/generative-art-nft#32]
What is the "edition" I'm being asked for supposed to be? I can't find any sort of information on what it is, or even where I should look to check what edition I'm supposed to be using.

**Comment by rounakbanik** [comment]
When you create artwork, you're asked to name the collection. The name of the collection is the edition.

**Comment by jjoh417** [comment]
This is wrong it is not the name of the collection...look at the medium article. The collection is called Scrappy Squirrels yet the edition they put was V2...Anyone else help with the edition?
